css/index.css

```diff
@@ -143,4 +243,34 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media(max-width: 600px) {
```

Comment:
Great work stretching with the media queries! I would recommend changing some things around with your container and navbar as well to remove the horizontal scrollbars, but I know the time was limited. Great work here!